src/Common/Common.Api/Reports/ReportsClient.cs

```diff
@@ -17,7 +19,79 @@ public ReportsClient(IPowerBIClient client) : base(client)
 
         public IEnumerable<Report> GetReports()
         {
-            return this.Client.Reports.GetReports().Value.Select(x => (Report)x);
+            return this.Client.Reports.GetReports().Value?.Select(x => (Report)x);
+        }
+
+        public IEnumerable<Report> GetReportsAsAdmin(string filter = null, int? top = null, int? skip = null)
+        {
+            return this.Client.Reports.GetReportsAsAdmin(filter: filter, top: top, skip: skip).Value?.Select(x => (Report)x);
+        }
+
+        public IEnumerable<Report> GetReportsAsAdminForWorkspace(Guid workspaceId, string filter = null, int? top = null, int? skip = null)
+        {
+            return this.Client.Reports.GetReportsAsAdmin(groupId: workspaceId.ToString(), filter: filter, top: top, skip: skip).Value?.Select(x => (Report)x);
+        }
+
+        public IEnumerable<Report> GetReportsForWorkspace(Guid workspaceId)
+        {
+            return this.Client.Reports.GetReports(groupId: workspaceId.ToString()).Value?.Select(x => (Report)x);
+        }
+
+        public Stream ExportReport(Guid reportId, Guid? workspaceId = default)
+        {
+            return workspaceId.HasValue && workspaceId.Value != default ?
+                this.Client.Reports.ExportReport(groupId: workspaceId.Value.ToString(), reportKey: reportId.ToString()) :
+                this.Client.Reports.ExportReport(reportKey: reportId.ToString());
+        }
+
+        public IEnumerable<Dashboard> GetDashboards()
```

Comment:
You can fix this at any time since it's not customer-facing like the module is, but I don't think Dashboards and Tiles should live under the Reports wrapper.

Reply:
Agree the models should probably be seperate to mimic the client

Reply:
Created issue #35